pwndbg/commands/nearpc.py

```diff
@@ -85,7 +85,7 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
 
     # Print out each instruction
     for i,s in zip(instructions, symbols):
-        asm    = pwndbg.disasm.color.instruction(i)
+        asm    = pwndbg.disasm.color.instruction(i, highlighted=i.address == pc)
```

Comment:
`pwndbg.disasm.color.instruction()` also has access to `pc`, so there's no need for the conditional here